File: Java/Array/SmallestNumberInArray.java
class SmallestNumberInArray {
    public static void main(String[] args) {
        int[] arr = {132, 24, 38,473,902,625,629,23,32};
        int SmallNum = Integer.MAX_VALUE;
        for(int i = 0; i< arr.length; i++)
        {
            SmallNum = Math.min(SmallNum,arr[i]);
        }
    System.out.println("The smallest number in the array is :" + SmallNum);
    }
}

File: Python/Array/SmallestNumberInArray.py
import sys # will provide access to some variable used or maintained by sys interpreter 
arr = list(map(int, input().strip().split()))
SmallNum = min(arr)
print(SmallNum)

# second smallest
secondSmallest = sys.maxsize #gives the largest element
for a in arr:
    if a < secondSmallest and a >SmallNum:
        secondSmallest = a 
print(secondSmallest)